src/pybind/mgr/dashboard/frontend/src/app/shared/directives/dimless-binary.directive.ts

```diff
@@ -86,6 +86,12 @@ export class DimlessBinaryDirective implements OnInit {
   setValue(value: string) {
     if (/^[\d.]+$/.test(value)) {
       value += this.defaultUnit || 'm';
+    } else {
+      if (value) {
+        this.control.control.setValue(value);
+        this.control.control.addValidators(Validators.pattern(/^[a-zA-Z\d. ]+$/));
```

Comment:
Don't we already have a validator for checking a valid number?

Reply:
but this checks number and alphabets along with a space.